getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,197 @@
+
+### Install Calico with the VPP dataplane on an EKS cluster
+
+#### Requirements
+
+For these instructions, we will use `eksctl` to provision the cluster. However, you can use any of the methods in {% include open-new-window.html text='Getting Started with Amazon EKS' url='https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html' %}
+
+Before you get started, make sure you have downloaded and configured the {% include open-new-window.html text='necessary prerequisites' url='https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html#eksctl-prereqs' %}
+
+#### Provision and configure the cluster
+
+1. First, create an Amazon EKS cluster without any nodes.
+
+   ```bash
+   eksctl create cluster --name my-calico-cluster --without-nodegroup
+   ```
+
+2. Since this cluster will use {{site.prodname}} for networking, you must delete the `aws-node` daemon set to disable AWS VPC networking for pods.
+
+   ```bash
+   kubectl delete daemonset -n kube-system aws-node
+   ```
+
+3. Now that you have a cluster configured, you can install {{site.prodname}}.
+
+   ```bash
+   kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp-eks.yaml
+   ```
+
+3. Finally, add nodes to the cluster.
+
+   ```bash
+   eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto --max-pods-per-node 100
+   ```
+
+   > **Tip**: Without the `--max-pods-per-node` option above, EKS will limit the {% include open-new-window.html text='number of pods based on node-type' url='https://github.com/awslabs/amazon-eks-ami/blob/master/files/eni-max-pods.txt' %}. See `eksctl create nodegroup --help` for the full set of node group options.
+   {: .alert .alert-success}
+
+#### (Optional) Switch to the DPDK driver
+
+The DPDK driver is supported on EKS, and will provide better networking performance. In order to enable it, follow the steps above, but use th following manifest at step 3:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp-eks-dpdk.yaml
+```
+
+### Install Calico with the VPP dataplane on any Kubernetes cluster
+
+#### Requirements
+
+The VPP dataplane has the following requirements:
+
+**Required**
+- A blank Kubernetes cluster, where no CNI was ever configured.
+- The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply, except those related to the management of `cali*`, `tunl*` and `vxlan.calico` interfaces.
+
+**Optional**
+
+The following may enable VPP to use more efficient drivers on some hardware:
+- At least 128 x 2MB-hugepages are available (`cat /proc/meminfo | grep HugePages_Free`)
+- The `vfio-pci` (named `vfio_pci` on centos) or `uio_pci_generic` kernel module is loaded
+
+These requirements can be satisfied with the following commands:
+````bash
+modprobe vfio-pci
+sysctl -w vm.nr_hugepages=128
```

Comment:
I think we spoke about changing this to:
```suggestion
echo "vfio-pci" > /etc/modules-load.d/95-vpp.conf
modprobe vfio-pci
echo "vm.nr_hugepages = 128" >> /etc/sysctl.conf
sysctl -p
```

Reply:
Thanks for the ping, forgot to push the changes, it should be good now ;-)